docs/source/techniques/ISISPowder-OSIRIS-v1.rst

```diff
@@ -0,0 +1,307 @@
+.. _isis-powder-diffraction-osiris-ref:
+
+=====================================================
+ISIS Powder Diffraction Scripts - OSIRIS Reference
+=====================================================
+
+.. contents:: Table of Contents
+    :local:
+
+.. _creating_osiris_object_isis-powder-diffraction-ref:
+
+Creating OSIRIS Object
+------------------------
+This method assumes you are familiar with the concept of objects in Python.
```

Comment:
would it be better to say something like:

This document assumes a familiarity with :ref:`intro_to_objects-isis-powder-diffraction-ref`